src/lib/DatePicker.svelte

```diff
@@ -427,4 +412,13 @@
       color: var(--date-picker-selected-color, inherit)
       background: var(--date-picker-selected-background, rgba(2, 105, 247, 0.2))
       border: 2px solid var(--date-picker-highlight-border, #0269f7)
+
+  .calendar-day:focus
+    outline: none
+  .calendar-day:focus-visible
+    transition: all 80ms cubic-bezier(0.4, 0.0, 0.2, 1)
+    border: 1px solid rgba(108, 120, 147, 0.3)
```

Comment:
Bug: the size of the popup changes when tabbing to the calendar

Reply:
I fixed this by adding a slight border around the grid that matches the style of the dropdowns. could not get it to retain the same size when it's transparent (my css foo isn't expert-level) but...I think it gives another visible cue that you can tab into it... 

![image](https://user-images.githubusercontent.com/2457369/155032812-788de5df-94ce-4169-96ad-38fafbaca4a4.png)

Reply:
You can make it happen by having the border still be `1px`, but using `transparent` as the color

Reply:
I'm actually digging the border for the reasons mentioned above - what do you think?

Reply:
I like how it makes the border not just be completely gone when you tab out of it, but at the same time it looks a bit rough when there's a border there with barely any padding around it

Reply:
How about a lighter border around the calendar, and more padding above the header row: 
![image](https://user-images.githubusercontent.com/39845713/155223168-2f0829e9-3301-428f-80e6-e259ca6dc1c8.png)

Reply:
Looks better, though I personally prefer it transparent. I'm still skeptical about merging this, so if you want to make a fork instead, then you can make your own decision for that